common/test/acceptance/tests/lms/test_certificate_web_view.py

```diff
@@ -1,24 +1,18 @@
 """
 Acceptance tests for the certificate web view feature.
 """
-from nose.plugins.attrib import attr
-
 from common.test.acceptance.fixtures.certificates import CertificateConfigFixture
-from common.test.acceptance.fixtures.course import CourseFixture, CourseUpdateDesc, XBlockFixtureDesc
+from common.test.acceptance.fixtures.course import CourseFixture
 from common.test.acceptance.pages.common.auto_auth import AutoAuthPage
 from common.test.acceptance.pages.lms.certificate_page import CertificatePage
-from common.test.acceptance.pages.lms.course_home import CourseHomePage
-from common.test.acceptance.pages.lms.courseware import CoursewarePage
-from common.test.acceptance.pages.lms.progress import ProgressPage
-from common.test.acceptance.pages.lms.tab_nav import TabNavPage
```

Comment:
Nice. Are these just leftovers from flaky test removals?

Reply:
Looks like: https://github.com/edx/edx-platform/pull/18168 .  They showed up in PyCharm as unused imports, and this should fix a few pylint warnings.